pymeasure/instruments/oxfordinstruments/base.py

```diff
@@ -23,7 +23,7 @@
 #
 
 
-from pymeasure.adapters import VISAAdapter
+from pymeasure.instruments import Instrument
 from pyvisa.errors import VisaIOError
 
```

Comment:
```suggestion
from pyvisa import constants as vconst
```
needed for the proper asrl options for parity and stop bits